doc/modules/neural_networks_supervised.rst

```diff
@@ -148,13 +162,21 @@ value is `1` represents the assigned classes of that sample::
     >>> clf.predict([0., 0.])
     array([[0, 1]])
 
+Following plot shows varying decision function depending on the value of alpha.
```

Comment:
Then we should add a new subsection titled "Regularization" that introduces weight decay parametrizable with the alpha constructor param and then include the that example to see the impact.